src/test/resources/app.properties

```diff
@@ -1,2 +1,3 @@
 chromeDriver = webdriver.chrome.driver
-chromeDriverPath = /Users/avoloshchuk/IdeaProjects/com.sharelane/src/test/resources/chromedriver79
\ No newline at end of file
+chromeDriverPath = src/test/resources/chromedriver79
```

Comment:
it's better but still not good. 
With the option implemented by you we need to keep the artifact inside **git** repo - but it's not necessary due to you already have it in you local **maven** repository as a downloaded dependency. 
So all you need - just modify the path to the driver starting it with some env property that pointed to the maven local repository and the particular driver artifact inside .

Reply:
The same about the particular Crome-driver.jar - I can't comment on binary files due to GitHub limitations so I place my comment here:
this file (Crome-driver.jar ) shouldn't be in the git repository at all

Reply:
Chromedriver setup is now delegated to WebDriverManager.